src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/HardCodeEncryptionKeySources.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities.PooledObjects;
+using static Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis.SourceInfo;
```

Comment:
>using static [](start = 0, length = 12)

Instead of "using static", don't make the delegates nested inside of `SourceInfo`.

(Personally, I'd go without "using static" because I'd rather than have more verbosity than more potential ambiguity.)